README.md

```diff
@@ -76,20 +76,22 @@ long_line_under_100_chars
 # Extra Specs Coding Style
 
 - Add a blank line between `let` and `before`.
+- Prefer `ModelKlass.new` over `build_stubbed` over `build` over `create` for optimal performance.
```

Comment:
Sometimes, I would prefer use `build_stubbed(:model_klass, :factory_trait)` instead of `ModelKlass.new(attr_1: value_1, attr_2: value_2, attr_3: value_3, attr_4: value_4)`.

The choice of readability over performance is most of the time related to the implementation context. I don't think this is something worth having in a coding style.

Reply:
Not enough context here, this needs extra explanation that factories might still be executing some persistence-related stuff  when using `build_stubbed` and in that case, when the data integrity and the state of valid attributes are needed (which would be the case for `build_stubbed`), `ModelKlass.new` might be a good alternative.